src/components/Goods/Goods.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react';
+import { createId } from '../../utils/uuid-creator';
+
+interface GoodsProps {
+  goods: string[];
+}
+
+export const Goods: FC<GoodsProps> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li key={createId()} data-cy="Good">
```

Comment:
I guess this function is a bit overkill, good will be enough here, but let it be as it is